packages/babel-parser/test/fixtures/core/object/invalid-setter/output.json

```diff
@@ -0,0 +1,52 @@
+{
+  "type": "File",
+  "start":0,"end":15,"loc":{"start":{"line":1,"column":0},"end":{"line":1,"column":15}},
+  "errors": [
+    "SyntaxError: setter must have exactly one formal parameter (1:3)"
```

Comment:
Unrelated, but I think we can improve this error message:

```
A setter must have exactly one parameter.
```

Or Typescript's:

```
A 'set' accessor must have exactly one parameter.
```